src/software/SfM/main_ComputeFeatures.cpp

```diff
@@ -219,7 +215,7 @@ int main(int argc, char **argv)
     Image<unsigned char> imageMask;
 
     const std::string sGlobalMask_filename = stlplus::create_filespec(sfm_data.s_root_path, "mask.png");
```

Comment:
Initially the SfM container path because sfm_data is the most "global" thing we could have :). But you can choose something else ultimately.